src/components/Svg/Icons/Bunny.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
This icon probably exist here https://pancakeswap.github.io/pancake-uikit/?path=/story/components-svg-icons--icons